core/client/app/controllers/settings/navigation.js

```diff
@@ -122,6 +122,14 @@ export default Controller.extend(SettingsSaveMixin, {
         addItem() {
             let newNavItem = this.get('newNavItem');
 
+            // If the url sent through is blank (user never edited the url)
+            if (newNavItem.get('url') === '') {
```

Comment:
By default all URLs are relative to the blog url (the url input component strips out the base url when notifying of a change) so I think to keep things consistent and simple it would be best to set the url to `/` here.